controllers/helpers_test.go

```diff
@@ -259,7 +259,7 @@ func TestReconcileAzureSecret(t *testing.T) {
 
 	cluster.Default()
 	azureCluster.Default()
-	azureCluster.ClusterName = "testCluster"
+	azureCluster.Name = "testCluster"
```

Comment:
I believe this should be `cluster.Name`

Reply:
basing this of the fact that we're using `clusterScope.ClusterName()` outside tests which maps to `cluster.Name`